modules/ipi-install-installing-rhel-on-the-provision-node.adoc

```diff
@@ -7,6 +7,4 @@
 
 = Installing RHEL on the provisioner node
 
-With the networking portions complete, the next step in installing the {product-title} cluster is to install RHEL 8.1 on the provisioner node.
-This node will be used as the orchestrator while installing the OCP cluster on the three Control Plane or master nodes and the at least two worker nodes.
-For the purposes of this document, installing RHEL on the provisioner node is out of scope. However, options include but are not limited to using a RHEL Satellite server, PXE, or installation media.
+With the networking configuration complete, the next step is to install RHEL 8.1 on the provisioner node. The installer uses the provisioner node as the orchestrator while installing the OCP cluster on the three control plane nodes and the at least two worker nodes. For the purposes of this document, installing RHEL on the provisioner node is out of scope. However, options include but are not limited to using a RHEL Satellite server, PXE, or installation media.
```

Comment:
s/OCP/{product-title}

Reply:
s/RHEL/{op-system-base}

We have this defined in the common-attributes.adoc file, as well as some other commonly referenced product names and other terms